src/services/item/repository.ts

```diff
@@ -92,13 +94,18 @@ export const ItemRepository = AppDataSource.getRepository(Item).extend({
       parsedExtra = { folder: { childrenOrder: [] } };
     }
 
+    const finalSettings = {
+      ...settings,
+      ...(settings.hasThumbnail === undefined ? DEFAULT_THUMBNAIL_SETTING : {}),
```

Comment:
Why not destructure before `...settings` in all cases? Are there cases where we don't want to add the default thumbnail props?
Actually I think if you do 
```
...DEFAULT_SETTINGS,
...settings
```
it should work as well, won't it?

Reply:
Yes it should, I will update it 👍